### PR TITLE
🐐 README changes to reflect Labs CI/CD being depricated 🐐

### PR DESCRIPTION
**Pull request by springdo**
#### What is this PR About?
Readme updates - nothing too crazy really :)

#### How do we test this?
1) Go to this PR
2) Read the Diff (wear 👓 if needs be )
3) Merge changes 

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,11 @@ A set of buildConfigs for building custom Jenkins images for OpenShift.
 ## Related Content
 
 * [Container Pipelines](https://github.com/redhat-cop/container-pipelines) - A set of Jenkins piplines for OpenShift
-* [Labs CI/CD](https://github.com/rht-labs/labs-ci-cd) - A comprehensive end to end pipeline using many modern testing and quality tools
+* [Labs Ubiquitous Journey](https://github.com/rht-labs/ubiquitous-journey) - A collection of ArgoCD apps for:
```
Comment (oybed) [inline README.md:63]: Could you add something like a footnote, etc. here around `f.k.a. Labs CI/CD`?

Comment (springdo) [reply README.md:63]: @oybed  - Do you want me to just keep Labs CI/CD in there and mention it as a good place to turn to for OCP v.3.X?

Comment (oybed) [reply README.md:63]: @springdo Sure, that sounds good.

Comment (springdo) [reply README.md]: Cool added both for the history books 
📖 👴

**Pull request merged**